README.md

```diff
@@ -4,6 +4,8 @@
 Practice space for using the Github Workflow.
 
 Participants:
+Colin Conway
```

Comment:
Don't really like the name here, change it to something better if possible

Reply:
whats your middle name? Trying to add those

Reply:
Looks fantastic, great work